src/pages/ProductList/ProductList.jsx

```diff
@@ -1,7 +1,180 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import styled from 'styled-components';
+import rArrow from '../../assets/images/right-arrow.png';
+import lArrow from '../../assets/images/left-arrow.png';
+import CardList from '../Mypage/components/Products/ProductList';
 
 const ProductList = () => {
-  return <div>ProductList</div>;
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [posts, setPosts] = useState([]);
+
+  const offset = searchParams.get('offset');
+  const limit = searchParams.get('limit');
+  const subCategory = searchParams.get('subCategory');
+
+  useEffect(() => {
+    fetch(
+      `/data/productList.json?offset=${offset}&limit=${limit}&status=${CATEGORY_SORT.map(
+        item => {
+          return { item: 'status' };
+        }
+      )}`
```

Comment:
```suggestion
    fetch(
      `/data/productList.json?offset=${offset}&limit=${limit}&status=&{status}`
```

왜 status를 map을 돌아서 구현하셨을까요?
해당 status에 맞는 쿼리를 요청하면 될 것 같습니다!